src/common/test-config.c

```diff
@@ -134,11 +133,10 @@ test_get_strvs (void)
 static void
 test_load_dir (void)
 {
-  cockpit_config_dirs[0] = SRCDIR "/src/ws/mock-config";
+  g_setenv("XDG_CONFIG_DIRS", "/var/lib/empty:" SRCDIR "/src/ws/mock-config", 1);
```

Comment:
The canonical non-existing path is /nonexisting; arguably it's very improbable that this exists, but let's rather not take a chance?

Reply:
ok